lib/install/management/create.go

```diff
@@ -148,12 +148,12 @@ func (d *Dispatcher) relocateAppliance() error {
 	// }
 	// hMoref := hosts[0].Reference()
 
-	host, err := randomPolicy.RecommendHost(d.op, d.appliance)
+	hosts, err := randomPolicy.RecommendHost(d.op, d.appliance.Session, []*object.HostSystem{d.appliance.Host})
```

Comment:
Should we instead use the snippet that's commented out above (lines 135-149)? It passes nil to `RecommendHost` to rank all hosts in the cluster.

Reply:
Good catch, fixed!